frontend/src/api/mockAPI/assignments.js

```diff
@@ -98,6 +98,19 @@ export class Assignment extends MockAPIController {
                 active_offer_url_token: activeOffer.url_token,
             });
         }
+        // Add position start and end dates if they are not defined
+        const position = this.getPosition(matchingAssignment)
+        if (!matchingAssignment.start_date) {
+            Object.assign(ret, {
+                start_date: position.start_date
+            });
```

Comment:
No need to use `Object.assign` here. Just `ret.start_date = ...`

Reply:
I was just following the same format as from lines 89 to 100, which seems to use Object.assign rather than individually assigning the attributes. Should we always use this format moving forwards? I've changed it to ret.start_date =  ...

Reply:
Yeah. Those lines are weird too and probably should be changed...